src/query/functions/temporal/base.go

```diff
@@ -49,15 +49,27 @@ type baseOp struct {
 // skipping lint check for a single operator type since we will be adding more
 // nolint : unparam
 func newBaseOp(args []interface{}, operatorType string, processorFn MakeProcessor) (baseOp, error) {
-	if operatorType != HoltWintersType && operatorType != PredictLinearType {
+	if operatorType != HoltWintersType && operatorType != PredictLinearType && operatorType != QuantileType {
```

Comment:
Seconding +arnikola's earlier comment here; why not handle this in the same way that we do `HoltWinters` and `PredictLinear`, and handle this casing in `NewFunctionExpr`? Handling the argument extraction in 2 places is unnecessarily confusing imo.